templates/CRM/Admin/Form/Setting/SettingField.tpl

```diff
@@ -2,7 +2,7 @@
 <tr class="crm-setting-form-block-{$setting_name}">
   <td class="label">{$form.$setting_name.label}</td>
   <td>
-    {if !empty($fieldSpec.wrapper_element)}
+    {if array_key_exists('wrapper_element', $fieldSpec) && !empty($fieldSpec.wrapper_element)}
```

Comment:
@eileenmcnaughton should the 2nd part be && array_key_exists(0, $fieldSpec,wrapper_element) rather than the !empty() for secure smarty reasons?

Reply:
@seamuslee001 hmm - I guess in practice it is either there & has array key 0 or not there - so I suspect the first check is all that is really actioned

Reply:
I'm ok with this but I just thought we were trying to get rid of the !emptys